modules/documents/client/views/partials/modal-document-confirm-publish.html

```diff
@@ -18,15 +20,21 @@ <h3 class="modal-title">Confirm Publish File(s)</h3>
 	</div>
 
 	<div class="modal-body"  ng-if="confirmDlg.missingCategory.length > 0">
-		These documents require a document category before they can be published
+		<div class="alert alert-danger" >
+			<span ng-if="confirmDlg.missingCategory.length === 1">This file requires a document category before it can be published:</span>
+			<span ng-if="confirmDlg.missingCategory.length > 1">These files require a document category before they can be published:</span>
+		</div>
 		<ul class="confirm-list">
 			<li ng-repeat="t in confirmDlg.missingCategory">{{t.displayName}}</li>
 		</ul>
 	</div>
 
 	<div class="modal-footer">
-		<button class="btn btn-default" type="button" ng-click="confirmDlg.cancel()">{{confirmDlg.cancelText}}</button>
+		<button class="btn btn-default" type="button" ng-click="confirmDlg.cancel()">
```

Comment:
Are you asking for "<div class="modal-body" ng-if="confirmDlg.errMsg && confirmDlg.missingCategory.length === 0">" to be changed to "<div class="modal-body" ng-show="confirmDlg.errMsg && confirmDlg.missingCategory.length === 0">"